ci/build_python_pypi.sh

```diff
@@ -8,11 +8,36 @@ python -m pip install build --user
 export GIT_DESCRIBE_TAG=$(git describe --abbrev=0 --tags)
 export GIT_DESCRIBE_NUMBER=$(git rev-list ${GIT_DESCRIBE_TAG}..HEAD --count)
 
+# Build date for PyPI pre-releases
+export RAPIDS_VERSION_NUMBER="23.08"
+export RAPIDS_FULL_VERSION_NUMBER="${RAPIDS_VERSION_NUMBER}.00"
```

Comment:
Can't hardcode the patch version (unless you also update via `update-version.sh` but we do not do this in any other repo).

Since during a release, `update-version.sh` is run before building, you could pull the current version from `pyproject.toml`. Then only adjust it to the pre-release version for not release builds.

Reply:
The version is being updated in https://github.com/rapidsai/dask-cuda/pull/1204/files#diff-8a0188e09c831b0e6ca8c1c51ab640efd1e519b1d7f94ab6251c9f0b90a82304R51 . I have no preference for getting the version from `pyproject.toml` or updating it with `update-version.sh` (except the latter is already done, and perhaps slightly easier). Do you have a strong preference for the `pyproject.toml` option?

Reply:
I have a weak pref for pyproject.toml, not a strong one. FWIW here's how we do it for our other wheels: https://github.com/rapidsai/gha-tools/blob/main/tools/rapids-pip-wheel-version